extensions/gitpod-remote/package.json

```diff
@@ -196,32 +251,65 @@
           "group": "remote_00_gitpod_navigation@130",
           "when": "gitpod.inWorkspace == true && gitpod.workspaceOwned == true && gitpod.workspaceShared == true"
         },
-        {
-          "command": "gitpod.openInStable",
-          "group": "remote_00_gitpod_navigation@900",
-          "when": "gitpod.inWorkspace == true && gitpod.UIKind == 'web'"
-        },
-        {
-          "command": "gitpod.openInInsiders",
-          "group": "remote_00_gitpod_navigation@1000",
-          "when": "gitpod.inWorkspace == true && gitpod.UIKind == 'web'"
-        },
         {
           "command": "gitpod.openInBrowser",
           "group": "remote_00_gitpod_navigation@1000",
           "when": "gitpod.inWorkspace == true && gitpod.UIKind == 'desktop'"
         }
       ]
+    },
+    "views": {
+      "portsView": [
+        {
+          "id": "gitpod.portsView",
+          "name": "",
+          "type": "webview",
+          "contextualTitle": "",
+          "icon": "$(plug)"
+        }
+      ]
+    },
+    "viewsContainers": {
+      "panel": [
+        {
+          "id": "portsView",
+          "title": "Exposed Ports",
```

Comment:
non-block: Change it to `Gitpod Ports` makes it more clear?

Reply:
No, it does not, ports are not belonging to Gitpod, they are the same ports but in different networks. We should point out to difference in networks. And exposed means exposed on Internet in our docs.